### PR TITLE
Enhancement/custom search polygon

### PR DESCRIPTION
**Pull request by lucianobarauna**
Sexy menu ;-)

### CODE REVIEW

File: src/components/GlobalFilter/GlobalFilter.js [hunk 1]
```diff
@@ -22,46 +36,60 @@ const GlobalFilter = ({
...
+                        <label htmlFor="searchField"><i className="fa fa-search"></i></label>
+                    </fieldset>
+                    <fieldset className="global-filter-form--selectiontype">
+                        <label>Tipo de seleção:</label>
+                        <label htmlFor="selectionType1" className="input-checkopacity">
+                            <input
+                                name="selectionType"
+                                type="radio"
+                                id="selectionType1"
+                                value="borda"
```
Comment (nighto) [inline src/components/GlobalFilter/GlobalFilter.js]: I think this `value` attribute was forgotten, no? Don't look like it makes a difference.

File: src/components/GlobalFilter/GlobalFilter.js [hunk 2]
```diff
@@ -22,46 +36,60 @@ const GlobalFilter = ({
...
+                                defaultChecked="checked"
+                                onClick={(e) => handleTypeChange(e)} />
+                            <span>Demarcada</span>
+                        </label>
+                        <label htmlFor="selectionType2" className="input-checkopacity">
+                            <input
+                                name="selectionType"
+                                type="radio"
+                                id="selectionType2"
+                                value="opaco"
```
Comment (nighto) [inline src/components/GlobalFilter/GlobalFilter.js]: The same for this one.

File: src/components/GlobalFilter/GlobalFilter.js (hunk 1 above)
Comment (lucianobarauna) [reply src/components/GlobalFilter/GlobalFilter.js]: Yes, it's right, I'll remove it.

File: src/components/GlobalFilter/GlobalFilter.js (hunk 2 above)
Comment (lucianobarauna) [reply src/components/GlobalFilter/GlobalFilter.js]: Ok

**Pull request merged**